### ISSUE TITLE
it worked for me

### ISSUE THREAD

**Issue by ivanpmartell** [MatthewHightech/stockSim1.0#2]
I followed the readme and I don't get the error, so it must be a setting in your computer. I got this:

`auth.service.ts:98 Document written with ID:  123`

Sorry, this is the only way I found to quickly communicate with you. Please check to see if the document was created. Hope a restart or something fixes it for you.

**Comment by MatthewHightech** [comment]
> I followed the readme and I don't get the error, so it must be a setting in your computer. I got this:
> 
> `auth.service.ts:98 Document written with ID: 123`
> 
> Sorry, this is the only way I found to quickly communicate with you. Please check to see if the document was created. Hope a restart or something fixes it for you.

@ivanpmartell Thanks for testing it! That's crazy that it worked, but I guess good? 

Unless you changed any code, that was just a read so it didn't change anything in the DB, but if you got that response, that's good news, and everything else should work. I'll do a wipe of my local copy, and just re-clone and install the whole repo. If it still doesn't work, then I don't know anymore :\

I'll let you know though. Thanks again for your help!

**Comment by MatthewHightech** [comment]
So @ivanpmartell  I tried it on my computer by cloning the repo from scratch and it still didn't work.
Then I made a brand new project from scratch and tried that and it still didn't work, until I waited 4 minutes, and then the data came through after my client making numerous failed requests. I've never seen this before, maybe @jaymichls if you have some time could give us a hand trying to figure this out? I posted a more thorough explanation of the issue on the HighTechU team portal [HERE](https://team.hightechu.ca/d/537-firebase-issue-not-connecting-to-firestore-using-angularfire6)
